s2a_options.go

```diff
@@ -64,6 +64,14 @@ func NewUID(name string) Identity {
 	return &uid{uid: name}
 }
 
+type VerificationModeType int
```

Comment:
Please add a top-level comment for this, e.g. "VerificationModeType specifies the mode that S2A must use to verify the peer certificate chain."

Reply:
Will do, thanks!